Public/Src/Engine/Dll/DirectoryScrubber.cs

```diff
@@ -360,6 +360,7 @@ public bool RemoveExtraneousFilesAndDirectories(CancellationToken cancellationTo
                         {
                             try
                             {
+                                Tracing.Logger.Log.ScrubbingDirectory(pm.LoggingContext, directory);

```

Comment:
>Tracing.Logger.Log.ScrubbingDirectory(pm.LoggingContext, directory); [](start = 32, length = 68)

Apparently we removed this logging to improve performance in this PR:
commit cc29aeea: Merged PR 389089: Directory Scrubber Perf Improvements ([https://dev.azure.com/mseng/_git/Domino/commit/cc29aeea3142e7aa2b8d2787c936458160eccb41?refName=refs%2Fheads%2Fmaster&_a=compare&path=%2FPublic%2FSrc%2FEngine%2FDomino.Engine%2FDirectoryScrubber.cs](https://dev.azure.com/mseng/_git/Domino/commit/cc29aeea3142e7aa2b8d2787c936458160eccb41?refName=refs%2Fheads%2Fmaster&_a=compare&path=%2FPublic%2FSrc%2FEngine%2FDomino.Engine%2FDirectoryScrubber.cs)) 

It's probably best to leave it out here to avoid perf regression.